.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on: [push]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare repository
+        # Fetch full git history and tags
+        run: git fetch --unshallow --tags
+
+      - name: Use Node.js 12.x
```

Comment:
we have .nvmrc using `10.16.0`

Reply:
Not anymore https://github.com/LandrAudio/linting-and-formatting/pull/50/files#diff-2c8effdf840690ccb88c7e21e56148978c6adb2c6926530c58ff0e47a9588b44R1

Reply:
cool ! by the way you could use the .nvmrc version. but it's not mandatory 😄 

https://github.com/actions/setup-node/issues/32#issuecomment-525791142